src/jquery.autofilter.js

```diff
@@ -63,15 +63,23 @@
         table._allColumnFilters = allColumnFilters;
     }
 
+    //Creating custom case insensitive expression for contains
+    jQuery.expr[':'].icontains = function(a, i, m) {
+        return jQuery(a).text().toUpperCase().indexOf(m[3].toUpperCase()) >= 0;
+    };
+
     $.fn.autofilter = function() {
         // setup the filter modal dialog
-        var modalWidgetId = "_jquery_autofilter_modal";
-        var modalWidget = $("#" + modalWidgetId);
+        var modalWidgetId = "_jquery_autofilter_modal",
+            modalWidget = $("#" + modalWidgetId),
+            $this = $(this),
```

Comment:
What's this line for?

Reply:
i wanted to save the plugin context, so i can use that later to send my custom event.